app/assets/javascripts/schools.js

```diff
@@ -5,11 +5,9 @@ $(document).ready(function(){
     event.preventDefault();
   });
   $("form .add_fields").click(function(event){
```

Comment:
`on` method is the preferred method nowadays instead of the actual event name. http://api.jquery.com/on/